css/style.css

```diff
@@ -220,34 +220,52 @@ main {
 }
 
 .toggle-switch {
-  display: flex;
+  display: inline-flex;
   align-items: center;
   justify-content: space-around;
   height: 30px;
-  width: 70px;
-  margin-left: 10px;
+  width: 55px;
+  margin-left: 5px;
   background-color: var(--secondary-color);
-  border-radius: 2px;
+  border-radius: 5px;
+  position: relative;
 }
 
-input[type="checkbox"] {
-  -moz-appearance: none;
-  margin: 0;
+.toggle-switch input{
+  display: none;
```

Comment:
Tried the accessibility fix. But I'm unable to tab the toggle-switch and activate it. The main area as a whole gets tab focus. Not sure why.
`.toggle-switch {
  display: inline-flex;
  align-items: center;
  justify-content: space-around;
  height: 30px;
  width: 55px;
  margin-left: 5px;
  background-color: var(--secondary-color);
  border-radius: 5px;
  position: relative;
  cursor: pointer;
}

.toggle-switch input {
  -moz-appearance: none;
  opacity: 0;
}

.toggle-switch:active {
  border:1px solid var(--button-active-border-color);
}`